File: com/pdfjet/Align.java
/**
 * Align.java
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
package com.pdfjet;

/**
 * Used to specify the text alignment in paragraphs.
 * See the Paragraph class for more details.
 */
public class Align {
    /** The default constructor */
    public Align() {
    }

    /** Specifies left alignment */
    public static final int LEFT    = 0x00000000;
    /** Specifies centered paragraph */
    public static final int CENTER  = 0x00100000;
    /** Specifies right alignment */
    public static final int RIGHT   = 0x00200000;
    /** Specifies justified paragraph */
    public static final int JUSTIFY = 0x00300000;

    /** Specifies top alignment */
    public static final int TOP     = 0x00400000;
    /** Specifies bottom alignment */
    public static final int BOTTOM  = 0x00500000;
}

File: src/align/align.go
package align

/**
 * align.go
 *
©2025 PDFjet Software

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.
*/

// Used to specify the text alignment in paragraphs.
// See the Paragraph class for more details.
const (
	Left    = 0x00000000
	Center  = 0x00100000
	Right   = 0x00200000
	Justify = 0x00300000
	Top     = 0x00400000
	Bottom  = 0x00500000
)
